Content.Server/NPC/Systems/NPCCombatSystem.Melee.cs

```diff
@@ -122,6 +122,13 @@ private void Attack(NPCMeleeCombatComponent component, TimeSpan curTime, EntityQ
             return;
         }
 
+        // Players cannot attack entities inside containers since melee context menu was removed, so mirror this for NPCs.
+        if (_containers.IsEntityInContainer(component.Target))
+        {
+            component.Status = CombatStatus.TargetUnreachable;
+            return;
+        }
+
```

Comment:
Doesn't this mean players can still attack targets in containers with malicious messages, seems more important to fix that.

Reply:
Hypothetically yes, I can take a look at that